src/test/resources/features/SalesForceBilling.feature

```diff
@@ -18,3 +18,18 @@ Feature: salesforce demo
     Examples:
       | TestData |
       | TestData1 |
+
+  @salesforceBill
+  Scenario Outline: salesforce demo
+    Given Read the test data  "<TestData>" from Excel
+    When Navigate to Salesforce as "BillingAnother Admin"
+    And Login to Salesforce with "BillingAnother Admin" credentials
+    And Click on Okay
+    And user clicks on the App launcher button
```

Comment:
Please follow the same pattern throughout the feature file, start the steps from capital letter